rust/Cargo.lock

```diff
@@ -1,7 +1,5 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
```

Comment:
I did not touch this file, let me know if this is expected

Reply:
I would be curious to see if the file is generated differently on macOS vs our docker container (linux).